render-wasm/src/render.rs

```diff
@@ -468,15 +468,19 @@ impl RenderState {
         ex + interest_delta
         ey + interest_delta
         */
+        let tile_center = ((ex - sx) / 2, (ey - sy) / 2);
         self.pending_tiles = vec![];
         self.surfaces.cache_clear_visited();
         for y in sy..=ey {
             for x in sx..=ex {
                 let tile = (x, y);
-                self.pending_tiles.push(tile);
+                let distance = tiles::manhattan_distance(tile, tile_center);
+                self.pending_tiles.push((x, y, distance));
                 self.surfaces.cache_visit(tile);
             }
         }
+        // reorder by distance to the center.
+        self.pending_tiles.sort_by(|a, b| b.2.cmp(&a.2));
```

Comment:
I guess we want to sort in ascending order, to process closer tiles first, right?

```suggestion
        self.pending_tiles.sort_by(|a, b| a.2.cmp(&b.2));
```